apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,146 @@
+    status: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative', 'white'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'grid',
+          placeItems: 'start',
+          gap: '20px',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <NotificationMarker {...args}>
+        <SvgNotification width={16} height={16} fill='currentColor' />
+      </NotificationMarker>
+      <NotificationMarker {...args}>Normal</NotificationMarker>
+    </>
+  );
+};
+
+Basic.args = {
+  status: 'primary',
+};
+
+export const WithButtons: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+
+      <Button styleType='borderless'>
+        <NotificationMarker {...args}>Inbox</NotificationMarker>
+      </Button>
+    </>
+  );
+};
+
+WithButtons.args = {
+  status: 'primary',
+};
+
+export const Pulsing: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='primary' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+
+      <NotificationMarker status='primary' {...args}>
+        <Text style={{ fontWeight: '700' }}>URGENT</Text>
+      </NotificationMarker>
+    </>
+  );
+};
+
+Pulsing.args = {
+  status: 'negative',
+  pulsing: true,
+};
+
+export const Status: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='primary' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='positive' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='warning' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='negative' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+
+      <NotificationMarker status='primary' {...args}>
+        Primary
+      </NotificationMarker>
+      <NotificationMarker status='positive' {...args}>
+        Positive
+      </NotificationMarker>
+      <NotificationMarker status='warning' {...args}>
+        Warning
+      </NotificationMarker>
+      <NotificationMarker status='negative' {...args}>
+        Negative
+      </NotificationMarker>
+    </>
+  );
```

Comment:
Maybe add an example for 'white' status in here? It'd probably be helpful visually to go along with explanation 
![image](https://user-images.githubusercontent.com/37936169/208144261-40c4614f-8176-4930-bfc2-9993d7f6e441.png)

Reply:
I found an example in `Button.stories.tsx` that has 'white' status. Even though it's an example there, I think it'd still be relevant here if you want to just copy it over or do something similar.

Reply:
I added examples for `styleType = white` in `NotificationMarker -> Status` story. I also changed the grid layout to make it more intuitive.

<img width="300px" src="https://user-images.githubusercontent.com/45748283/208152725-4c7d19c4-96d0-4c49-b3a0-a09d270b8104.png" />

Reply:
Looks good; I like it!